src/app/views/quests/view/view.route.ts

```diff
@@ -1,7 +0,0 @@
-import { RouteRecordRaw } from 'vue-router';
-
-export const routeQuestsView: RouteRecordRaw = {
```

Comment:
do we have a redirect rule for these? we have links to at least the quests log before